### ISSUE TITLE
The contrast is wrong, everything is hard to read

### ISSUE THREAD

**Issue by gregkare** [travis-ci/travis-web#344]
On the current version every piece text is too light. I'm 32 years old and am not visually impaired. The online tools I have found that check the contrast for compliance with [WCAG 2.0](http://www.w3.org/TR/WCAG20/) don't work with JavaScript websites, so I'll link to a simpler tool instead:

http://webaim.org/resources/contrastchecker/

The shades of grey used for text (for example #969496 on #fff, #808080 on #FAFAF8),  are definitely not legible on those backgrounds. In fact they're so light they would be more legible on a black background.

![screen shot 2015-03-25 at 12 42 45](https://cloud.githubusercontent.com/assets/43297/6823861/93cb8fd4-d2ec-11e4-8816-31e4adb16cd6.png)

**Comment by joshk** [comment]
Thanks for creating the ticket.

Could you explain a bit further which areas of Travis you find the hardest to read? 

Top bar? Build header? Job list? Side bar?

**Comment by saltinejustine** [comment]
Thank you @gregkare for your feedback and the screenshot is super handy. We're working on rolling out slight changes so please stay tuned. We're definitely listening!

**Comment by gregkare** [comment]
Everything is equally hard to read, it also makes it really hard to tell which are the most important parts. On light backgrounds the most important pieces of information should be darker so you instantly see the hierarchy of information.

Thanks

**Comment by saltinejustine** [comment]
Thanks @gregkare. We're definitely still working on the visuals of travis as a whole. Please keep the feedback coming as you see things. We look forward to it!

**Comment by joshk** [comment]
We do use different text colours, for example the grey for the job list, and commit message in the build header, and the text in the top bar, is darker than the right hand side text in the build header.

If would be very handy to us if you could rank the contrast issues on the screen.

Thanks.

**Comment by lucaspinto** [comment]
For me, the hardest parts to read are the "duration" and "finished" bits.

**Comment by saltinejustine** [comment]
@lucaspinto thanks so much for that specific feedback. very helpful to know what is a problem for users. <3

**Comment by joshk** [comment]
We have pushed some recent updates which should address these issues.

Closing this issue for now.

Thank you!
